src/Crypto/Lol/Applications/SymmSHE.hs

```diff
@@ -276,7 +276,7 @@ switch hint c = rescaleLinearMSD $ untag $ knapsack <$>
 
 -- | Constraint synonym for key switching.
 type KeySwitchCtx gad t m' zp zq zq' =
-  (ToSDCtx t m' zp zq,
+  (ToSDCtx t m' zp zq, NFData (Cyc t m' zq'),
```

Comment:
GHC wanted "NFData (CRTExt zq')".  CRTExt isn't in scope in SymmSHE, and I didn't want to import CRTrans just for this constraint. Import aside, the next thing GHC would ask for is "NFData zq'". I *could* use the primitive constraints, but it seemed cleaner to use the one for Cyc.